### PR TITLE
Add helpful error when blacklist and whitelist are empty arrays.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

File: lib/addons/should-include.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 export default function shouldIncludeAddon(addon, blacklist, whitelist) {
+  if (blacklist && blacklist.lenght === 0 && whitelist && whitelist.length === 0) {
```
Comment (dgeb) [inline lib/addons/should-include.js]: s/lenght/length

**Comment by dgeb** [comment]
@rwjblue looks good except for that typo, and the fact that tests pass _with_ the typo makes me think that another test is probably needed.

**Comment by rwjblue** [comment]
Pretty wat RE: the typo + tests pass, will dig in...

**Comment by locks** [comment]
Ah 💩 , I saw the inline comment before seeing this thread.

**Comment by Turbo87** [comment]
closing due to inactivity

**Pull request closed without merging**